airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -177,6 +177,37 @@ def _make_safe_label_value(self, input_value: str | datetime) -> str:
             return pod_generator.datetime_to_label_safe_datestring(input_value)
         return pod_generator.make_safe_label_value(input_value)
 
+    def get_pod_labels_combined_str_to_pod_map(self) -> dict[str, k8s.V1Pod]:
+        """
+        List the worker pods owned by this scheduler and create a map containing pod combined labels search str -> pod.
+
+        For every pod, it creates two below entries in the map
```

Comment:
Two entries? Where is the second?

Reply:
Sorry, we were added with execution_date as well. However, that was deprecated and removed in this PR. I will update the comment too.

Reply:
I have updated the comment.